prod/status_im/i18n_resources.cljs

```diff
@@ -0,0 +1,49 @@
+(ns status-im.i18n-resources
```

Comment:
Is there a way to minimize the difference between dev and prod code or is it going to be like this for every mobule?

Reply:
for translations there is no way, but this PR doesn't introduce class modules. For cljs modules I will create a macro which will encapsulate the difference and we will not need to put any extra code to `prod` and `dev`. But again, this PR is a different case.